### DISCUSSION TITLE
想问一下如何添加多栏，同时期待付费版，感谢大神

### DISCUSSION THREAD

**Discussion by Tuz10** [Tokinx/Wing#90] [讨论]
![15143e06c5f02c1c71013426f279358](https://user-images.githubusercontent.com/104152099/206069575-35e5e996-4011-4a25-919e-d8c7fe6cd18c.jpg)
如何添加该样式

**Comment by LingJueYa** [comment]
左边是wp的菜单 = 后台- 外观- 菜单
右边是在输入框内 ，以“ # ”的格式写入

**Reply by Tuz10** [reply]
谢谢解答，初次上手，很多不懂，感谢

**Reply by rluoluo** [reply]
> 左边是wp的菜单 = 后台- 外观- 菜单 右边是在输入框内 ，以“ # ”的格式写入

怎么写入啊？具体是哪个输入框？希望大佬能回复下多谢

**Reply by noteMay** [reply]
> > 左边是wp的菜单 = 后台- 外观- 菜单 右边是在输入框内 ，以“ # ”的格式写入
> 
> 怎么写入啊？具体是哪个输入框？希望大佬能回复下多谢

笔记那边输入就好了……
